### DISCUSSION TITLE
大家使用期间有没有遇到过翻译后cpu占用过高的问题

### DISCUSSION THREAD

**Discussion by WvvvWv** [immersive-translate/immersive-translate#494] [Help]
### 概述 | Summary

如下图所示，我只开了一个github issus页面全局翻译了一下，一直就是占用过高。
![image](https://github.com/immersive-translate/immersive-translate/assets/40822202/c27559fd-4645-4f46-a784-39a05a46ee2b)

**Comment by versun** [comment]
@WvvvWv 
可以打开浏览器自带的任务管理器，查看具体是什么在占用CPU哈
![image](https://github.com/immersive-translate/immersive-translate/assets/2398708/c2bfe3cd-f9fe-4423-86d1-12744772f37b)

**Reply by WvvvWv** [reply]
第一个图是未开启翻译，第二个图是开启翻译后
![image](https://github.com/immersive-translate/immersive-translate/assets/40822202/c1d16df9-602d-4196-bc28-df8396120906)
![image](https://github.com/immersive-translate/immersive-translate/assets/40822202/f5789ede-8d06-40fa-bcfe-3489f072888d)

**Reply by versun** [reply]
看截图不确定是不是因为本插件造成的CPU占用，我这边测试是正常的，本插件也没有任何需要后台常驻运行的任务。
你可以自己排查下截图中的2个插件哈，或者也可以尝试更新下浏览器和插件哈
![image](https://github.com/immersive-translate/immersive-translate/assets/2398708/9174da6c-d03a-47d3-8fa8-2560d2f57eb1)

**Reply by WvvvWv** [reply]
> 看截图不确定是不是因为本插件造成的CPU占用，我这边测试是正常的，本插件也没有任何需要后台常驻运行的任务。 你可以自己排查下截图中的2个插件哈，或者也可以尝试更新下浏览器和插件哈 ![image](https://user-images.githubusercontent.com/2398708/241156997-9174da6c-d03a-47d3-8fa8-2560d2f57eb1.png)
查到了，是图中这个URL in title和翻译插件冲突导致CPU占用飙升，但具体原因不清楚，卸载掉URL in title就好了。
